swagger/event-manager.yaml

```diff
@@ -647,6 +761,10 @@ definitions:
       name:
         type: string
         maxLength: 32
+      kind:
+        type: string
+        pattern: '^[\w\d\-]+$'
```

Comment:
really no reason that we have a pattern on a read-only field... no biggie though.